Portal/View/Shadow.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  Shadow.swift
+//  Portal
+//
+//  Created by Cristian Ames on 7/11/17.
+//  Copyright © 2017 Guido Marucci Blas. All rights reserved.
+//
+import UIKit
```

Comment:
We should not depend on UIKit at this level. UIKit specific should be part of the renderer inside the UIKit directory